### DISCUSSION TITLE
How to tear down a gridjs table?

### DISCUSSION THREAD

**Discussion by nabsabraham** [grid-js/gridjs#1380] [General]
I want to be able to destroy the gridjs table so I can ask the server for a fresh/different list of data to display in the table, is that possible? My stack is python/flask. For comparision, I am currently using DataTables and I have access to this [functionality](https://datatables.net/reference/api/destroy())

**Comment by leehrat** [comment]
If you're populating the table using something like this:

`server: {
        url: '/your_url_to_get_data',
        then: results => results.data,
        total: results => results.total,
        },`

Then you can use:

`grid.forceRender();`

in whatever function that you want triggering the fresh data need. That basically calls your '/your_url_to_get_data' again to get the data.

**Reply by nabsabraham** [reply]
hmm the issue is my url changes on button click because it gets a new parameter for a GET request: `url: '/your_url_to_get_data?param=<something_new>` -- so to have this work, I could put the code to generate the table in my button event handler and everytime the button is clicked, the `param` changes and a new table is created?
